basic/23-erc865-and-erc875/testAccounts.json

```diff
@@ -0,0 +1,12 @@
+[
```

Comment:
Put these into .env file would be better I think.

Reply:
yeah, but need to do some string operations cause there are many privates keys.